_episodes/03-infrastructure.md

```diff
@@ -184,10 +184,14 @@ are modifying.
 
 ## Style Maintenance
 
-The template for Carpentry lessons is managed through the [styles repository](https://github.com/swcarpentry/styles), which is
-maintained by [Raniere Silva](https://github.com/rgaiacs) in
-coordination with the Lesson Infrastructure Subcommittee. If you
-are interested in being involved, please contact Raniere.
+The template for Carpentries lessons is managed through the [styles
+repository](https://github.com/swcarpentry/styles), which is maintained by the
+[Lesson Infrastructure Committee](https://carpentries.org/lesson-infra/). The
+activity of the Committee was put on hold during the Covid-19 pandemic but is
```

Comment:
Using all capitals is how the AP Style and [WHO have used it](https://www.who.int/emergencies/diseases/novel-coronavirus-2019/technical-guidance/naming-the-coronavirus-disease-(covid-2019)-and-the-virus-that-causes-it)

```suggestion
activity of the Committee was put on hold during the COVID-19 pandemic but is
```